Client/Frontend/Strings.swift

```diff
@@ -1386,16 +1386,13 @@ extension String {
 
 // MARK: - Credential Provider
 extension String {
-    public static let WelcomeViewTitle = MZLocalizedString("WelcomeView.Title", tableName: "CredentialProvider", comment: "Label displaying welcome view title")
-    public static let WelcomeViewSpinnerSyncingLogins = MZLocalizedString("WelcomeView.Spinner.SyncingLogins", tableName: "CredentialProvider", comment: "Label for syncing your logins spinner")
-    public static let WelcomeViewSpinnerDoneSyncingLogins = MZLocalizedString("WelcomeView.Spinner.DoneSyncingLogins", tableName: "CredentialProvider", comment: "Label for finishing syncing your logins")
-    public static let LoginsListSearchCancel = MZLocalizedString("LoginsList.Search.Cancel", tableName: "CredentialProvider", comment: "Cancel button title")
-    public static let LoginsListSearchPlaceholder = MZLocalizedString("LoginsList.Search.Placeholder", tableName: "CredentialProvider", comment: "Placeholder text for search field")
-    public static let LoginsListSelectPasswordTitle = MZLocalizedString("LoginsList.SelectPassword.Title", tableName: "CredentialProvider", comment: "Label displaying select a password to fill instruction")
-    public static let LoginsListNoMatchingResultTitle = MZLocalizedString("LoginsList.NoMatchingResult.Title", tableName: "CredentialProvider", comment: "Label displayed when a user searches and no matches can be found against the search query")
-    public static let LoginsListNoMatchingResultSubtitle = MZLocalizedString("LoginsList.NoMatchingResult.Subtitle", tableName: "CredentialProvider", comment: "Label that appears after the search if there are no logins available to this account")
-    public static let LoginsListNoLoginsFoundTitle = MZLocalizedString("LoginsList.NoLoginsFound.Title", tableName: "CredentialProvider", comment: "Label informing the user the account has no logins available")
-    public static let LoginsListNoLoginsFoundSubtitle = MZLocalizedString("LoginsList.NoLoginsFound.Subtitle", tableName: "CredentialProvider", comment: "Label shown when there are no logins to list")
+    public static let WelcomeViewTitle = MZLocalizedString("WelcomeView.Title", value: "Take your passwords everywhere", comment: "Label displaying welcome view title")
```

Comment:
@rilury Can you please rename this to `LoginsWelcomeViewTitle` then it is a bit less generic and clear that it is part of the same set of strings for logins.

Reply:
Sure, I will rename it to LoginsWelcomeViewTitle 👍